### PR TITLE
download IPs from private github repo

### PR DESCRIPTION
**Pull request by AndyBoWu**
* hide github authorization token
* fetch IP from github instead from s3
* code refactoring

### CODE REVIEW

File: devops/bin/update_server_list.bash [hunk 1]
```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+source /Users/bwu2/.github/token.sh
```
Comment (LeoHChen) [inline devops/bin/update_server_list.bash]: parameterize it please.

File: devops/bin/update_server_list.bash [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+declare -a ip_array0
+declare -a ip_array1
+declare -a ip_array2
+declare -a ip_array3
+
+# retrieve host file
+download_hosts_file () {
+
+	# download list of IP for shard 0
+	curl -H "Authorization: token $git_token" \
```
Comment (LeoHChen) [inline devops/bin/update_server_list.bash]: use a for loop

File: devops/bin/update_server_list.bash [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+
+}
+
+
+update_prometheus_config () {
+
+	echo "[*] Overwritting prometheus.yml .. "
+	# sudo scp -i $path_keys/california-key-benchmark.pem prometheus.yml ec2-user@ec2-52-52-168-89.us-west-1.compute.amazonaws.com:/etc/prometheus/prometheus.yml
+	scp -i $path_keys/california-key-benchmark.pem prometheus.yml ec2-user@ec2-52-52-168-89.us-west-1.compute.amazonaws.com:~/
```
Comment (LeoHChen) [inline devops/bin/update_server_list.bash]: is there a DNS name for Prometheus server?

File: devops/bin/update_server_list.bash (hunk 2 above)
Comment (AndyBoWu) [reply devops/bin/update_server_list.bash]: wrote the script in a rush. yes, should be done in a for a loop.

File: devops/bin/update_server_list.bash (hunk 1 above)
Comment (AndyBoWu) [reply devops/bin/update_server_list.bash]: hmm, right, it sounds like a better design. will do.

**Comment by AndyBoWu** [comment]
integrated with other project

**Pull request closed without merging**